catalyst/dl/utils/swa.py

```diff
@@ -0,0 +1,75 @@
+from typing import List
+from collections import OrderedDict
+import glob
+import os
+from pathlib import Path
+
+import torch
+
+
+def average_weights(state_dicts: List[dict]) -> OrderedDict:
+    """
+    Averaging of input weights.
+
+    Args:
+        state_dicts (List[dict]): Weights to average
```

Comment:
could you please remove type from docs (the type already specified in arguments)